### PR TITLE
Update prep

### PR DESCRIPTION
**Pull request by somexpert**
Fixes flake8 errors and adds more testing environments.

### CODE REVIEW

**Comment by somexpert** [comment]
ping @ldko

File: nomination/views.py [hunk 1]
```diff
@@ -1,70 +1,84 @@
...
+    )
+
 
 def get_project(slug):
-    #get the project by the project slug
+    # get the project by the project slug
     try:
         project = Project.objects.get(project_slug=slug)
-    except:
+    except Project.DoesNotExist, Project.MultipleObjectsReturned:
```
Comment (ldko) [inline nomination/views.py]: I don't think we need to catch `Project.MultipleObjectsReturned`.

File: nomination/url_handler.py [hunk 2]
```diff
@@ -113,112 +117,124 @@ def validate_date(dateinput):
...
+
 def add_url(project, form_data):
     summary_list = []
     form_data['url_value'] = check_url(form_data['url_value'])
-    #Get the system nominator
+    # Get the system nominator
     try:
         system_nominator = Nominator.objects.get(id=settings.SYSTEM_NOMINATOR_ID)
-    except:
+    except Nominator.DoesNotExist, Nominator.MultipleObjectsReturned:
```
Comment (ldko) [inline nomination/url_handler.py]: I don't think you could get `Nominator.MultipleObjectsReturned` here.

Comment (somexpert) [reply nomination/url_handler.py]: oh right, primary key

File: nomination/url_handler.py [hunk 3]
```diff
@@ -85,26 +90,25 @@ def handle_metadata(request, posted_data):
...
+        '%b %d %Y', '%b %d, %Y',                # 'Oct 25 2006', 'Oct 25, 2006'
+        '%d %b %Y', '%d %b, %Y',                # '25 Oct 2006', '25 Oct, 2006'
+        '%B %d %Y', '%B %d, %Y',                # 'October 25 2006', 'October 25, 2006'
+        '%d %B %Y', '%d %B, %Y',                # '25 October 2006', '25 October, 2006'
     )
     validdate = None
-    if dateinput != None and \
-      len(dateinput) != 0 and \
-      not dateinput.isspace():
+    if dateinput is not None and len(dateinput) != 0 and not dateinput.isspace():
```
[the remark this answers is not in the document]
Comment (somexpert) [reply nomination/url_handler.py]: since I've already touched this line I don't mind simplifying it to that

File: nomination/url_handler.py (hunk 2 above)
Comment (somexpert) [reply nomination/url_handler.py]: Yeah I'll remove that from here and the other places you pointed out.

File: nomination/url_handler.py [hunk 4]
```diff
@@ -382,89 +406,94 @@ def create_json_browse(slug, url_attribute, root=''):
...
-                        domain_name+'</a>',
-                                   'id': domain_name+','
-                                  }
-            #if the domain dictionary isn't empty
+                    domain_dict = {'text': '<a href=\"surt/(' +
+                                   domain_name+'\">' +
+                                   domain_name+'</a>',
+                                   'id': domain_name+','}
+            # if the domain dictionary isn't empty
             if len(domain_dict) > 0:
```
[the remark this answers is not in the document]
Comment (somexpert) [reply nomination/url_handler.py]: I think we should leave _most_ of these kinds of fixes for a later time when we clean up the code a bit more, but I don't mind doing this one now.

**Comment by ldko** [comment]
Ok :+1:

**Pull request merged**